docs/utils/data.rst

```diff
@@ -0,0 +1,215 @@
+`~astropy.utils.data.import_download_cache` to load the astropy cache from such a
+zipfile.
+
+Using Astropy With Limited or No Internet Access
+================================================
+
+You might want to use astropy on a telescope control machine behind a strict
+firewall. Or you might be running contionuous integration on your Astropy
+server and want to avoid hammering astronomy servers on every pull request for
+every architecture. Or you might not have access to US government or military
+web servers. Whichever is the case, you may need to avoid Astropy needing data
+from the Internet. There is no simple and complete solution to this problem at
+the moment, but there are tools that can help.
+
+Exactly which external data your project depends on will depend on what parts
+of Astropy you use and how. The most general solution is to use a computer that
+can access the Internet to run a version of your calculation that pulls in all
+the data files you will require, including sufficiently up-to-date versions of
+files like the IERS data that update regularly. Then once the cache on this
+connected machine is loaded with everything necessary, transport the cache
+contents to your target machine by whatever means you have available, whether
+by copying via an intermediate machine, sneakernet, or carrier pigeon. The
+cache directory itself is somewhat portable between machines of the same UNIX
+flavour; this may be sufficient if you can persuade your CI system to cache the
+directory between runs. For greater portability, though, you can simply use
+`~astropy.utils.data.export_download_cache` and
+`~astropy.utils.data.import_download_cache`, which are portable and will allow
+adding files to an existing cache directory.
+
+If your application needs IERS data specifically, you can download the
+appropriate IERS table, covering the appropriate time span, by any means you
+find convenient. You can then load this file into your application and use the
+resulting table rather than `~astropy.utils.iers.IERS_Auto`. In fact, the IERS
+B table is small enough that a version (not necessarily recent) is bundled with
+Astropy as ``astropy.utils.iers.IERS_B_FILE``. Using a specific non-automatic
+table also has the advantage of giving you control over exactly which version
+of the IERS data your application is using. See also :ref:`iers-working-offline`.
+
+If your issue is with certain specific servers, even if they are the ones
+Astropy normally uses, if you can anticipate exactly which files will be needed
+(or just pick up after Astropy fails to obtain them) and make those files
+available somewhere else, you can request they be downloaded to the cache
+using `~astropy.utils.data.download_file` with the ``sources`` argument set
+to locations you know do work.
+
+If you have a particular URL that is giving you trouble, you can download it
+using some other tool (for example ``wget``), possibly on another machine, and
+then use `import_to_cache`.
```

Comment:
you need to use the full API reference to make API links otherwise the docs build is failing due to missing sphinx links
```suggestion
then use `~astropy.utils.data.import_to_cache`.
```

Reply:
Aha! Thank you.